examples/fastify-typescript-starter/README.md

```
@@ -0,0 +1,73 @@
+```
+
+### Other Commands
+
+```bash
+npm run typecheck  # Check for TypeScript errors
+npm run format     # Format code with Prettier
+```
+
+## Project Structure
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)